### PR TITLE
Uploaded image scale to screen, zoom buttons

### LINKED ISSUE #101: Bug: PDF/Bilde scales ikke til fit to canvas
I dag: 

image

Sånn det burde se ut: 

image

Her er PDF for testing
[Jaktkart-Lunner-Almenning (1).pdf](https://github.com/TheAtlasRepository/ImgPDF2Map/files/14918673/Jaktkart-Lunner-Almenning.1.pdf)

### PR DESCRIPTION
**Pull request by sebmid**
# Changes

ZoomButtons,tsx **New**
+ New buttons for controlling zoom
+ also reset zoom button

moveImage.tsx
+ Now uses a scale factor to scale the image based on the size of the image
+ then does transform to center the image after the scaling has been done

splitview.tsx
+ just adds zoombuttons to component

### CODE REVIEW

File: frontend/components/ui/ZoomButtons.tsx
Comment (ostepizza) [inline frontend/components/ui/ZoomButtons.tsx:1]: Preferably use Button-component 😿

**Review by ostepizza — CHANGES_REQUESTED** [review]
pls look comment sorry 😭

File: frontend/components/ui/ZoomButtons.tsx
Comment (sebmid) [reply frontend/components/ui/ZoomButtons.tsx:1]: oki 🥉

**Review by ostepizza — APPROVED** [review]
"real man doing a pog-face"
image

**Review by Trivinyx — APPROVED** [review]
Goood

**Pull request merged**